src/storage/metrics.rs

```diff
@@ -43,6 +57,16 @@ lazy_static! {
         exponential_buckets(0.0005, 2.0, 20).unwrap()
     )
     .unwrap();
+    pub static ref SCHED_RAW_READ_HISTOGRAM_VEC: SchedDurationVec = {
+        register_static_histogram_vec!(
+            SchedDurationVec,
+            "tikv_scheduler_raw_read_command_duration_seconds",
```

Comment:
em, do you introduce another new metric name here? If yes, please use the old one.

/cc @breeswish

Reply:
here, I will change to use the old one.